rawkv/rawkv.go

```diff
@@ -504,6 +504,47 @@ func (c *Client) ReverseScan(ctx context.Context, startKey, endKey []byte, limit
 	return
 }
 
+// Checksum do checksum of continuous kv pairs in range [startKey, endKey).
+// If endKey is empty, it means unbounded.
+// If you want to exclude the startKey or include the endKey, push a '\0' to the key. For example, to scan
+// (startKey, endKey], you can write:
+// `Scan(ctx, push(startKey, '\0'), push(endKey, '\0'), limit)`.
```

Comment:
Seem it's the comments from `ReverseScan`.

Reply:
yes, they accept the same inpu parameters.

Reply:
```suggestion
// Checksum do checksum of continuous kv pairs in range [startKey, endKey).
// If endKey is empty, it means unbounded.
// If you want to exclude the startKey or include the endKey, push a '\0' to the key. For example, to checksum
// (startKey, endKey], you can write:
// `Checksum(ctx, push(startKey, '\0'), push(endKey, '\0'), limit)`.
```